Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/dotnet/sdk:5.0-buster-slim AS restore
+FROM mcr.microsoft.com/dotnet/sdk:6.0 AS restore
```

Comment:
Only curiosity. Why did you remove the distribution?

Reply:
Because mcr.microsoft.com/dotnet/sdk:6.0-buster-slim does not exist in https://hub.docker.com/_/microsoft-dotnet

Reply:
It is because `buster-slim` is an old distribution. I suppose that the updated equivalent wold be `bullseye-slim`.